CuitService.Test/TaxInfoApiTest.cs

```diff
@@ -146,7 +146,7 @@ public async Task GET_taxinfo_by_cuit_should_make_honor_original_format()
 
             // Assert
             // Validate indentation
-            Assert.Matches(@"^{\n\s+""(.+\n)*}$", content);
+            Assert.Matches(@"(?<=\"")([^\s,].*?)(?=\"")|null", content);
```

Comment:
What does it mean?

Reply:
It validates a correct JSON format

Reply:
Is it validating the indentation?

I mean, the previous assertion validated that the result was indented and splitter in more than a line, this new validation is different.

Reply:
Restored the test in #89